extensions/schema-compare/src/schemaCompareResult.ts

```diff
@@ -266,10 +266,6 @@ export class SchemaCompareResult {
 	}
 
 	public async execute(): Promise<void> {
-		if (this.schemaCompareOptionDialog && this.schemaCompareOptionDialog.deploymentOptions) {
```

Comment:
What happens in following
1. Open SCMP (say updated option A)
2. Open Options and change option B
Now when compare is executed the both Option A and B changes should be in effect.

Reply:
Both changes to option A and B will take effect. The options dialog is opened with the deployment options loaded from the scmp, then the deployment options are set to whatever the options dialog's options are when the dialog is closed